lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,9 @@
+namespace :auth_source_external do
+  desc 'Creates an external authentication source named "External", if no external authentication source exists.'
+  task :create => :environment do
+    User.as_anonymous_admin do
+      AuthSourceExternal.create!(name: 'External') unless AuthSourceExternal.any?
+      puts "Failed to create external authentication source.  External authentication source named '#{AuthSourceExternal.pluck(:name)}' with id '#{AuthSourceExternal.pluck(:id)}' is already present."
```

Comment:
this is still printed even if the auth source was created successfully. 
Perhaps it would be easier to do something along the lines of:
```ruby 
if AuthSourceExternal.any?
  puts 'failed...'
else
  create
  puts 'success..'
end
```